extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/standalone/VertxRequestHandler.java

```diff
@@ -97,7 +97,16 @@ private void dispatch(RoutingContext routingContext, InputStream is, VertxOutput
             VertxHttpResponse vertxResponse = new VertxHttpResponse(request, dispatcher.getProviderFactory(),
                     request.method(), allocator, output);
             // client address may not be available with VirtualHttp
-            SocketAddress socketAddress = request.remoteAddress();
+            SocketAddress socketAddress = null;
+            try {
+                socketAddress = request.remoteAddress();
+            } catch (NullPointerException npe) {
+                // ideally there shouldn't be an exception for this call.
+                // This is just to workaround a current issue in vertx
+                // https://github.com/quarkusio/quarkus/issues/5247
```

Comment:
can we also add the link to the vertx issue eclipse-vertx/vert.x#3181?

Reply:
@machi1990, Done